### DISCUSSION TITLE
[Warning]: Node-ipc package was sabotaged by maintainer.

### DISCUSSION THREAD

**Discussion by ACodingGenie** [darkreader/darkreader#8466] [General]
Not sure if the package is used within this application but `node-ipc` was sabotaged by the maintainer as a way to “protest” against the ongoing war. Please DO NOT update as it deletes all files on system once executed. Thank you.

URL: https://github.com/RIAEvangelist/node-ipc

**Comment by Gusted** [comment]
I had a quick grep trough package-lock.json doesn't seem like we have it as a dependency.
